metadata.json

```diff
@@ -19,37 +19,56 @@
   ],
   "operatingsystem_support": [
     {
-      "operatingsystem": "RedHat"
+      "operatingsystem": "RedHat",
+      "operatingsystemrelease": [
```

Comment:
the whole PR has a lot of changes. I suggest to pull some of them out into their own PRs. For example this metadata.json adjustment. It drops support for multiple OSses and also adds new ones if I read the diff correctly. I think that deserve its own PR.